lib/devise_security_extension/hooks/session_lockable.rb

```diff
@@ -4,11 +4,25 @@ def self.extract_email(env)
   rescue
     nil
   end
+
+  def self.extract_default_scope(env)
+    env['warden'].config.default_scope.safe_constantize
```

Comment:
It will be the value of the first 'devise_for' within the app. For us that will be 'users'.